server/routes/testutils/testData.ts

```diff
@@ -227,6 +228,14 @@ export default class TestData {
     actionedBy,
   })
 
+  static isExcludeDateDto = ({
+    isExcluded = true,
+    excluded = true,
+  }: Partial<IsExcludeDateDto> = {}): IsExcludeDateDto => ({
+    isExcluded,
+    excluded,
+  })
+
```

Comment:
I don't think this type is right - I think it's just got one property, `isExcluded`? But, tbh, I don't think it's worth bothering with a `TestData` for it as it is basically just a boolean.